src/products/models/link-product.ts

```diff
@@ -0,0 +1,15 @@
+import { Type } from 'class-transformer';
+import { ExtensionAttributes } from '../../common-models';
+
+
+export class LinkProduct{
+
+  sku: string = '';
+  link_type: string = '';
```

Comment:
`link_type` can be like:
```typescript
link_type:'crosssell'|'upsell'|'related'|'associated';
```

This would make it only accept those specified terms as values. You can also create a type alias like:
```typescript
export type LinkType = 'crosssell'|'upsell'|'related'|'associated';
```
Now you can directly use: `link_type: LinkType;`